### ISSUE TITLE
[Feature] Detect features like welded screws

### ISSUE THREAD

**Issue by luzpaz** [shaise/FreeCAD_SheetMetal#96]
One of the todos in `SheetMetalUnfolder.py` 
https://github.com/shaise/FreeCAD_SheetMetal/blob/4bd6542f41195936886ffd02385a0ad0ee43d77d/SheetMetalUnfolder.py#L62

**Comment by luzpaz** [comment]
Hi @jaisekjames  
is this on your radar?

**Comment by luzpaz** [comment]
@jaisekjames can you weigh-in?

**Comment by jaisekjames** [comment]
No, I am not working on this issue. I don't much Idea on unfolder coding.

Just an Idea.

We can select face that having this kind of deformation that cannot be unfolded. 

In Unfolder script, we can skip related faces that are inside that face so that same shape inside face is used.
![forming](https://user-images.githubusercontent.com/44542605/116516677-8198fc80-a8eb-11eb-8e3c-fdde3dd2ffa2.jpg)

In above image, If we can input skip face as top face & all deformed shape are inside that face. If we skip face without unfolding Unfolder will not fail & we will correct shape.

Only thing, we have to manually select faces required to skip.

**Comment by luzpaz** [comment]
Anyone interested in tackling this ? :point_up:  @Syres916 ?
